scripts/mellanox_db_migrator.py

```diff
@@ -0,0 +1,327 @@
+import syslog
```

Comment:
can we call this `mlnx_buffer_migrator.py`? I believe all other generic ones would still be part of `db_migrator.py`. This naming might give the impression that for all migration for mellanox platform , it would be this file.

Reply:
Sounds reasonable.
Fixed.